nms/app/components/AccountSettings.js

```diff
@@ -117,7 +117,11 @@ export default function AccountSettings() {
             />
           </AltFormField>
         </List>
-        <Button onClick={onSave} disabled={!isSaveEnabled}>
+        <Button
+          onClick={onSave}
+          disabled={!isSaveEnabled}
+          variant="contained"
+          color="primary">
```

Comment:
_Style question:_ Would it be possible and more aligned to the above `PasswordInput` to do the following? Or is this uncommon?
```javascript
        <Button
          onClick={onSave}
          disabled={!isSaveEnabled}
          variant="contained"
          color="primary"
        >
```

Reply:
Yes, this is done by the auto formatter (prettier). 
We could change the setting https://prettier.io/docs/en/options.html#bracket-line
But I'm not sure if should. You will get used it :)